pkg/agent/util/iptables/iptables.go

```diff
@@ -184,47 +185,60 @@ func matchProtocol(ipt *iptables.IPTables, protocol Protocol) bool {
 	return false
 }
 
-// DeleteRule checks if target rule already exists, deletes the rule if found.
-func (c *Client) DeleteRule(table string, chain string, ruleSpec []string) error {
-	for idx := range c.ipts {
-		exist, err := c.ipts[idx].Exists(table, chain, ruleSpec...)
+// DeleteRules checks if target rule already exists in all iptables, deletes the rule if found.
```

Comment:
"in all tables" in the updated comment is not correct. The original comment is correct

Reply:
Updated.